### PR TITLE
PIC-311: Added unpaid work to convictions response

### PR DESCRIPTION
**Pull request by katie-cawthorne**
- Added unpaid work to sentence within the convictions response, in line with community-api
- Updated tests
- Updated mock data

### CODE REVIEW

File: src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/UnpaidWork.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import io.swagger.annotations.ApiModel;
+import lombok.Builder;
+import lombok.Getter;
+
+@ApiModel("UnpaidWork")
+@Getter
+@Builder
+public class UnpaidWork {
+    private Integer offered;
```
Comment (jhevans) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/UnpaidWork.java:11]: We should document that this value is given in minutes... in fact maybe just rename it to minutesOffered and not bother with documenting it. Sorry I appreciate this is what it says in the ticket!

File: src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiAppointment.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CommunityApiAppointment {
```
Comment (jhevans) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiAppointment.java:9]: Maybe rename this? `CommunityApiAttendance` or something like that might be clearer?  Up to you

File: src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/UnpaidWork.java (hunk 1 above)
Comment (katie-cawthorne) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/UnpaidWork.java:11]: I agree, I did think that when I was doing it actually. I'll update it.

File: src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiAppointment.java (hunk 2 above)
Comment (katie-cawthorne) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiAppointment.java:9]: Sure makes more sense, I'll update it now.

Comment (katie-cawthorne) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiAppointment.java:9]: Actually, just had a look and we already have `CommunityApiAttendance` so I'll keep it as `CommunityApiAppointment`, it's actually referred to as `appointment` in community-api.

File: src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/UnpaidWork.java (hunk 1 above)
Comment (jhevans) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/UnpaidWork.java]: Awesome! Don't feel like the ticket is gospel, that was just my best interpretation of what seemed sensible at the time but we should feel free to change it if we find things that can be done better. Just leave a comment or something so people coming after know.

**Pull request merged**